src/service/translate.js

```diff
@@ -952,6 +954,50 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
   };
 
   /**
+     * @ngdoc function
+     * @name pascalprecht.translate.$translateProvider#postProcess
+     * @methodOf pascalprecht.translate.$translateProvider
+     *
+     * @description
+     * In some very special cases, a developer or translation agency might need an easy way
+     * to see the contexts, they're translating content for. To mark the keys and translations
+     * inside a page, the postProcessor gives the application developer the possibility to
+     * have such a behavior
+     *
+     * @param {Function=} fn Function to be called after the translation value has been set / resolved. The function
+     * itself will enrich every value being processed and then continue the normal resolver process
+     */
+  this.postProcess = function (fn) {
+    if (fn) {
+      postProcessFn = fn;
+    } else {
+      postProcessFn = undefined;
+    }
+    return this;
+  };
+
+  /**
+       * @ngdoc function
+       * @name pascalprecht.translate.$translateProvider#postProcessEnabled
+       * @methodOf pascalprecht.translate.$translateProvider
+       *
+       * @description
+       * Additional function to trigger the functionality for the postProcessing (enabling / disabling)
+       * This is useful if you want to enable / disable this feature depending on the environment / stage
+       * your application is running on
+       *
+       * @param {boolean} sets the postProcess function to enabled or disabled (true / false) or a default false
+       */
+  this.postProcessEnable = function (flag) {
+    if (flag === undefined) {
+      $postProcessEnabledState = false;
+    } else {
+      $postProcessEnabledState = flag;
+    }
+    return this;
```

Comment:
Why do we need tracking both states? If this could be changed at runtime I would understand.. but this this are one-time configuration things. Either there is a post processing function, or there is no one.